components/hierarchical-view/hierarchical-view-mixin.js

```diff
@@ -76,14 +76,18 @@ export const HierarchicalViewMixin = superclass => class extends superclass {
 	connectedCallback() {
 		super.connectedCallback();
 
-		this._isChildView();
+		this.__isChildView();
+
+		/* If this node is not yet known to be a childView (e.g., if there is a timing issue and the
```

Comment:
By "parent node exists"... is the issue that the parent node may not be connected yet?

Reply:
Yeah the issue is that it isn't connected yet. Agreed - there does seem like there should be a better way but I have not yet found it. The other "solution" that seemed to work was in menu-item https://github.com/BrightspaceUI/core/blob/master/components/menu/menu-item-mixin.js#L59 when it detects that it has a child view, set childview on `this.__children[0]`, since that would be the hierarchical view, but that just didn't seem great either.